### PR TITLE
Add algorithm to rotate images

### PR DESCRIPTION
**Pull request by matheustguimaraes**

### CODE REVIEW

File: digital_image_processing/rotation/rotation.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        get_rotation(gray_img, pts1, pts2, img_rows, img_cols),
+        get_rotation(gray_img, pts2, pts3, img_rows, img_cols),
+        get_rotation(gray_img, pts2, pts4, img_rows, img_cols),
+    ]
+
+    # plot different image rotations
+    fig = plt.figure(1)
+    titles = ["Original", "Rotation 1", "Rotation 2", "Rotation 3"]
+    for i in range(0, len(images)):
+        plt.subplot(2, 2, i + 1), plt.imshow(images[i], "gray")
```
Comment (cclauss) [inline digital_image_processing/rotation/rotation.py]: for i, image in enumerate(images):
    plt.subplot(2, 2, i + 1), plt.imshow(image, "gray")

File: digital_image_processing/rotation/rotation.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+from matplotlib import pyplot as plt
+from numpy import float32
+import cv2
+
+
+def get_rotation(img, pt1, pt2, rows, cols):
```
Comment (cclauss) [inline digital_image_processing/rotation/rotation.py]: Type hints?  https://docs.python.org/3/library/typing.html

File: digital_image_processing/rotation/rotation.py (hunk 1 above)
Comment (matheustguimaraes) [reply digital_image_processing/rotation/rotation.py]: Added enumerate in loop.

File: digital_image_processing/rotation/rotation.py (hunk 2 above)
Comment (matheustguimaraes) [reply digital_image_processing/rotation/rotation.py]: Added type hints in get_rotation(). Thanks for the help!

**Pull request merged**